index.html

```diff
@@ -84,197 +46,249 @@ <h4>Zero-ops OpenStack on just about any Linux box.</h4>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Secure</h3>
+            <div class="p-matrix__desc">
+              <p>
+                Runs safely on your laptop with state of the art isolation.
+              </p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Upstream</h3>
+            <div class="p-matrix__desc">
+              <p>
+                Pure upstream OpenStack delivered to your laptop.
+              </p>
+            </div>
           </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Complete</h3>
+            <div class="p-matrix__desc">
+              <p>
+                Includes all key OpenStack components: Keystone, Nova, Neutron, Glance, and Cinder.
+              </p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Featureful</h3>
+            <div class="p-matrix__desc">
+              <p>
+                All the cool things you probably want to try on a small, standard OpenStack are all built-in.
+              </p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Small</h3>
+            <div class="p-matrix__desc">
+              <p>
+                Use MicroStack in your CI/CD pipelines and get on with your day without headaches.
+              </p>
+            </div>
+          </div>
+        </li>
+      </ul>
+    </div>
+  </section>
 
-          <p>
-            Command <code>snap</code> not found? <a class="p-link--external" href="https://snapcraft.io/docs/core/install">Install snapd first</a>.
-          </p>
+  <section class="p-strip is-bordered" id="get-started" role="tablist">
+    <div class="u-fixed-width">
+      <h2>Get started with MicroStack</h2>
+      <div class="p-notification--information">
+        <p class="p-notification__response">
+          <span class="p-notification__status">Information:</span>
+          MicroStack requires at least 8 GB of RAM and a multi-core processor.
+        </p>
+      </div>
+      <h3>What OS are you developing on?</h3>
+    </div>
 
-          <h4>On Windows and macOS</h4>
+    <div class="row u-equal-height u-sv3">
+      <a id="tab-one" href="#tab-one__content"
+        class="col-small-2 col-medium-2 col-4 p-card p-tab u-align--center u-vertically-center js-tab" role="tab">
+        <img src="https://assets.ubuntu.com/v1/53c3be42-linuc-logo.svg" style="height: 90px; margin-top: 1rem;" alt="">
+        <h4 class="p-heading--five p-tab__title">Linux</h4>
```

Comment:
Accessibility oddity: That “Linux” happens to be outside an image, while “Windows” and “macOS” are `alt` text, is not a reason for “Linux” to be the only one that’s a heading. Either all three should be headings (e.g. `<h4`…`><img`…`alt="Windows"></h4>`), or none of them should be.